staging/src/k8s.io/apiserver/pkg/server/deleted_kinds.go

```diff
@@ -122,7 +135,7 @@ type removedInterface interface {
 
 // removeDeletedKinds inspects the storage map and modifies it in place by removing storage for kinds that have been deleted.
 // versionedResourcesStorageMap mirrors the field on APIGroupInfo, it's a map from version to resource to the storage.
-func (e *resourceExpirationEvaluator) removeDeletedKinds(groupName string, versionedResourcesStorageMap map[string]map[string]rest.Storage) {
+func (e *resourceExpirationEvaluator) RemoveDeletedKinds(groupName string, versionedResourcesStorageMap map[string]map[string]rest.Storage) {
```

Comment:
Why is this a  method? If it was not a method, but a simple func, it would not have to show up in upper interface.

Reply:
> Why is this a method? If it was not a method, but a simple func, it would not have to show up in upper interface.

The rules for applying it are contained in the evaluator.

Reply:
`e` is not used.

Reply:
missed `e.shouldServe`